docs/source/contributing/writing-docs.adoc

```diff
@@ -121,8 +121,8 @@ $ make clean_docs
 
 The {project} documentation is deployed on *docs.openshift.org* under link:https://docs.openshift.org/latest/minishift[https://docs.openshift.org/latest/minishift].
 
-To integrate with *docs.openshift.org*, we deliver a tarball that contains the {project} AsciiDoc files as well as some link:http://www.asciibinder.org/[AsciiBinder] metadata files.
-This tarball is generated with a Jenkins job on link:https://ci.centos.org/job/minishift-docs[CentOS CI] and is triggered as part of a xref:./releasing.adoc#cut-release[release].
+To integrate with *docs.openshift.org*, we deliver a tarball at link:http://artifacts.ci.centos.org/minishift/minishift/docs/latest/[artifact server] that contains the {project} AsciiDoc files as well as some link:http://www.asciibinder.org/[AsciiBinder] metadata files.
```

Comment:
Replace "tarball" with "compressed archive" or similar. Also, I think it would be cleaner to introduce the location of the archive in a separate sentence. Example:

```adoc
To integrate with *docs.openshift.org*, we deliver a compressed archive that contains the {project} AsciiDoc files as well as some link:http://www.asciibinder.org/[AsciiBinder] metadata files.
This archive is generated by a link:https://ci.centos.org/job/minishift-release[release job] which is triggered by an xref:../releasing.adoc#automated-release[automated release].
The generated archive is located on the link:http://artifacts.ci.centos.org/minishift/minishift/docs/latest/[artifact server].
```